src/components/SingleTrikot.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import {List, Image,Container, Flag} from "semantic-ui-react";
+import {List, Image,Container, Flag, Feed} from "semantic-ui-react";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'Container'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)